diff --git a/app/views/layouts/user_mailer.html.erb b/app/views/layouts/user_mailer.html.erb
index d196d35312..c82310ddf0 100644
--- a/app/views/layouts/user_mailer.html.erb
+++ b/app/views/layouts/user_mailer.html.erb
@@ -35,8 +35,8 @@ See docs/COPYRIGHT.rdoc for more details.
       }
 
       body {
-        font-family: Verdana, sans-serif;
-        font-size: 0.8em;
+        font-family: Calibri, Verdana, sans-serif;
+        font-size: 1.05em;
         color: #484848;
         background: #FFFFFF;
       }
@@ -68,8 +68,13 @@ See docs/COPYRIGHT.rdoc for more details.
         padding-left: 0.6em;
       }
       .footer {
+        color: #808080;
+        background: #FFFFFF;
+      }
+      .header {
         font-size: 0.8em;
-        font-style: italic;
+        color: #C4C4C4;
+        background: #FFFFFF;
       }
       a.user-mention:before {
         content: '@';
@@ -82,7 +87,6 @@ See docs/COPYRIGHT.rdoc for more details.
       <%= call_hook(:view_layouts_mailer_html_before_content, self.assigns) %>
       <%= yield %>
       <%= call_hook(:view_layouts_mailer_html_after_content, self.assigns) %>
-      <hr />
       <span class="footer"><%= OpenProject::TextFormatting::Renderer.format_text(Setting.localized_emails_footer) %></span>
   </body>
 </html>
