diff --git a/node_modules/@ng-select/ng-select/fesm2022/ng-select-ng-select.mjs b/node_modules/@ng-select/ng-select/fesm2022/ng-select-ng-select.mjs
index 8dcf1f5..d4ca72d 100644
--- a/node_modules/@ng-select/ng-select/fesm2022/ng-select-ng-select.mjs
+++ b/node_modules/@ng-select/ng-select/fesm2022/ng-select-ng-select.mjs
@@ -3500,6 +3500,9 @@ class NgSelectComponent {
     }
     _changeSearch(searchTerm) {
         this._searchTerm.set(searchTerm);
+        if (this.typeahead()?.observed) {
+            this.typeahead().next(searchTerm ?? '');
+        }
     }
     _scrollToMarked() {
         if (!this.isOpen() || !this.dropdownPanel()) {
