diff --git a/lib/plugins/acts_as_searchable/lib/acts_as_searchable.rb b/lib/plugins/acts_as_searchable/lib/acts_as_searchable.rb
index 9d0099d912..a69945948b 100644
--- a/lib/plugins/acts_as_searchable/lib/acts_as_searchable.rb
+++ b/lib/plugins/acts_as_searchable/lib/acts_as_searchable.rb
@@ -91,7 +91,7 @@ module Redmine
 
             token_clauses = columns.map { |column| "(LOWER(#{column}) LIKE ?)" }
 
-            if EnterpriseToken.allows_to?(:attachment_filters) && OpenProject::Database.allows_tsv?
+            if OpenProject::Database.allows_tsv?
               tsv_clauses = tsv_columns.map do |tsv_column|
                 OpenProject::FullTextSearch.tsv_where(tsv_column[:table_name],
                                                       tsv_column[:column_name],
