diff --git a/app/helpers/application_helper.rb b/app/helpers/application_helper.rb
index a4d22e91cb..c9c42e3658 100644
--- a/app/helpers/application_helper.rb
+++ b/app/helpers/application_helper.rb
@@ -306,7 +306,8 @@ module ApplicationHelper
       css << 'action-' + params[:action]
     end
 
-    css << "ee-banners-#{EnterpriseToken.show_banners? ? 'visible' : 'hidden'}"
+    # Only until remove completely Enterprise banners
+    css << "ee-banners-hidden"
 
     # Add browser specific classes to aid css fixes
     css += browser_specific_classes
