diff --git a/modules/auth_plugins/lib/open_project/plugins/auth_plugin.rb b/modules/auth_plugins/lib/open_project/plugins/auth_plugin.rb
index 5097d9bd30..5628530019 100644
--- a/modules/auth_plugins/lib/open_project/plugins/auth_plugin.rb
+++ b/modules/auth_plugins/lib/open_project/plugins/auth_plugin.rb
@@ -71,11 +71,7 @@ module OpenProject::Plugins
     def self.filtered_strategies(options)
       options.select do |provider|
         name = provider[:name]&.to_s
-        next true if !EnterpriseToken.show_banners? || name == 'developer'
-
-        warn_unavailable(name)
-
-        false
+        next true 
       end
     end
 
@@ -92,13 +88,6 @@ module OpenProject::Plugins
 
       [camelization, name].compact.first.underscore.to_sym
     end
-
-    def self.warn_unavailable(name)
-      RequestStore.fetch("warn_unavailable_auth_#{name}") do
-        Rails.logger.warn { "OmniAuth SSO strategy #{name} is only available for Enterprise Editions." }
-        true
-      end
-    end
   end
 
   class ProviderBuilder
