diff --git a/lib/open_project/configuration.rb b/lib/open_project/configuration.rb
index ed240d13f8..fcb63da256 100644
--- a/lib/open_project/configuration.rb
+++ b/lib/open_project/configuration.rb
@@ -120,7 +120,7 @@ module OpenProject
 
       'youtube_channel' => 'https://www.youtube.com/c/OpenProjectCommunity',
 
-      'ee_manager_visible' => true,
+      'ee_manager_visible' => false,
 
       # Health check configuration
       'health_checks_authentication_password' => nil,
@@ -174,10 +174,10 @@ module OpenProject
       'sentry_dsn' => nil,
       # Allow separate error reporting for frontend errors
       'sentry_frontend_dsn' => nil,
-      'sentry_host' => 'https://sentry.openproject.com',
+      'sentry_host' => 'https://sentry.com',
 
       # Allow connection to Augur
-      'enterprise_trial_creation_host' => 'https://augur.openproject.com',
+      'enterprise_trial_creation_host' => '',
 
       # Allow override of LDAP options
       'ldap_auth_source_tls_options' => nil,
