diff --git a/app/controllers/custom_fields_controller.rb b/app/controllers/custom_fields_controller.rb
index f76b869e6e..0267641574 100644
--- a/app/controllers/custom_fields_controller.rb
+++ b/app/controllers/custom_fields_controller.rb
@@ -118,9 +118,7 @@ class CustomFieldsController < ApplicationController
   def get_custom_field_params
     custom_field_params = permitted_params.custom_field
 
-    if !EnterpriseToken.allows_to?(:multiselect_custom_fields)
-      custom_field_params.delete :multi_value
-    end
+    custom_field_params.delete :multi_value
 
     custom_field_params
   end
