diff --git a/app/helpers/custom_styles_helper.rb b/app/helpers/custom_styles_helper.rb
index 2029c34af0..53c7f5d311 100644
--- a/app/helpers/custom_styles_helper.rb
+++ b/app/helpers/custom_styles_helper.rb
@@ -29,19 +29,18 @@
 #++
 
 module CustomStylesHelper
-  def apply_custom_styles?(skip_ee_check: OpenProject::Configuration.bim?)
+  def apply_custom_styles?
     # Apply custom styles either if EE allows OR we are on a BIM edition with the BIM theme active.
-    CustomStyle.current.present? &&
-      (EnterpriseToken.allows_to?(:define_custom_style) || skip_ee_check)
+    CustomStyle.current.present?
   end
 
   # The default favicon and touch icons are both the same for normal OP and BIM.
   def apply_custom_favicon?
-    apply_custom_styles?(skip_ee_check: false) && CustomStyle.current.favicon.present?
+    CustomStyle.current.favicon.present?
   end
 
   # The default favicon and touch icons are both the same for normal OP and BIM.
   def apply_custom_touch_icon?
-    apply_custom_styles?(skip_ee_check: false) && CustomStyle.current.touch_icon.present?
+    CustomStyle.current.touch_icon.present?
   end
 end
