diff --git a/app/controllers/custom_styles_controller.rb b/app/controllers/custom_styles_controller.rb
index 425f1cf0be..eed1d6c556 100644
--- a/app/controllers/custom_styles_controller.rb
+++ b/app/controllers/custom_styles_controller.rb
@@ -32,7 +32,6 @@ class CustomStylesController < ApplicationController
   menu_item :custom_style
 
   before_action :require_admin, except: [:logo_download, :favicon_download, :touch_icon_download]
-  before_action :require_ee_token, except: [:upsale, :logo_download, :favicon_download, :touch_icon_download]
   skip_before_action :check_if_login_required, only: [:logo_download, :favicon_download, :touch_icon_download]
 
   def show
@@ -132,12 +131,6 @@ class CustomStylesController < ApplicationController
     CustomStyle.current || CustomStyle.create!
   end
 
-  def require_ee_token
-    unless EnterpriseToken.allows_to?(:define_custom_style)
-      redirect_to custom_style_upsale_path
-    end
-  end
-
   def custom_style_params
     params.require(:custom_style).permit(:logo, :remove_logo, :favicon, :remove_favicon, :touch_icon, :remove_touch_icon)
   end
