diff --git a/app/views/projects/filters/_form.html.erb b/app/views/projects/filters/_form.html.erb
index 92210700ef..a455ae4016 100644
--- a/app/views/projects/filters/_form.html.erb
+++ b/app/views/projects/filters/_form.html.erb
@@ -83,11 +83,5 @@
         <%= submit_tag t('button_apply'), class: 'button -small -highlight', name: nil %>
       </li>
     </ul>
-    <% unless EnterpriseToken.allows_to?(:custom_fields_in_projects_list)%>
-      <enterprise-banner-bootstrap
-        text-message="<%= t('ee.upsale.project_filters.description_html') %>"
-        referrer="project-list-filter">
-      </enterprise-banner-bootstrap>
-    <% end %>
   </fieldset>
 <% end %>
