diff --git a/app/models/query/highlighting.rb b/app/models/query/highlighting.rb
index f26a721141..588b65f829 100644
--- a/app/models/query/highlighting.rb
+++ b/app/models/query/highlighting.rb
@@ -69,8 +69,6 @@ module Query::Highlighting
     end
 
     def highlighted_attributes
-      return [] unless EnterpriseToken.allows_to?(:conditional_highlighting)
-
       val = super
 
       if val.present?
@@ -81,8 +79,6 @@ module Query::Highlighting
     end
 
     def highlighting_mode
-      return :none unless EnterpriseToken.allows_to?(:conditional_highlighting)
-
       val = super
 
       if val.present?
