diff --git a/modules/grids/lib/grids/configuration/in_project_base_registration.rb b/modules/grids/lib/grids/configuration/in_project_base_registration.rb
index 4ed219b0b2..f3e4a3d0d8 100644
--- a/modules/grids/lib/grids/configuration/in_project_base_registration.rb
+++ b/modules/grids/lib/grids/configuration/in_project_base_registration.rb
@@ -24,8 +24,7 @@ module Grids::Configuration
     }
 
     queries_permission_and_ee_lambda = ->(user, project) {
-      save_or_manage_queries_lambda.call(user, project) &&
-        EnterpriseToken.allows_to?(:grid_widget_wp_graph)
+      save_or_manage_queries_lambda.call(user, project) 
     }
 
     view_work_packages_lambda = ->(user, project) {
