diff --git a/app/controllers/members_controller.rb b/app/controllers/members_controller.rb
index b88657dc5e..95d7f9fc14 100644
--- a/app/controllers/members_controller.rb
+++ b/app/controllers/members_controller.rb
@@ -221,7 +221,7 @@ class MembersController < ApplicationController
     user_ids.map do |id|
       if id.to_i == 0 && id.present? # we've got an email - invite that user
         # only admins can invite new users
-        if current_user.admin? && enterprise_allow_new_users?
+        if current_user.admin?
           # The invitation can pretty much only fail due to the user already
           # having been invited. So look them up if it does.
           user = UserInvitation.invite_new_user(email: id) ||
@@ -235,10 +235,6 @@ class MembersController < ApplicationController
     end.compact
   end
 
-  def enterprise_allow_new_users?
-    !OpenProject::Enterprise.user_limit_reached? || !OpenProject::Enterprise.fail_fast?
-  end
-
   def each_comma_seperated(array, &block)
     array.map { |e|
       if e.to_s.match /\d(,\d)*/
