diff --git a/app/views/members/_member_form.html.erb b/app/views/members/_member_form.html.erb
index d45024ff89..50a0e8c276 100644
--- a/app/views/members/_member_form.html.erb
+++ b/app/views/members/_member_form.html.erb
@@ -76,13 +76,6 @@ See docs/COPYRIGHT.rdoc for more details.
         </div>
       </div>
     </div>
-    <% if OpenProject::Enterprise.user_limit_reached? %>
-        <div class="notification-box -warning icon-warning" id="member-user-limit-warning" style="display: none;">
-          <div class="notification-box--content">
-            <p><%= I18n.t(:warning_user_limit_reached, upgrade_url: OpenProject::Enterprise.upgrade_path).html_safe %></p>
-          </div>
-        </div>
-    <% end %>
 <% end %>
 
 
