diff --git a/app/helpers/projects_helper.rb b/app/helpers/projects_helper.rb
index 39bde1981b..35e52759ee 100644
--- a/app/helpers/projects_helper.rb
+++ b/app/helpers/projects_helper.rb
@@ -52,7 +52,7 @@ module ProjectsHelper
       Queries::Projects::Filters::NameAndIdentifierFilter,
       Queries::Projects::Filters::TypeFilter
     ]
-    whitelist << Queries::Filters::Shared::CustomFields::Base if EnterpriseToken.allows_to?(:custom_fields_in_projects_list)
+    whitelist << Queries::Filters::Shared::CustomFields::Base 
 
     whitelist.detect { |clazz| filter.is_a? clazz }
   end
