diff --git a/app/services/users/register_user_service.rb b/app/services/users/register_user_service.rb
index ac7047a077..748606259d 100644
--- a/app/services/users/register_user_service.rb
+++ b/app/services/users/register_user_service.rb
@@ -37,7 +37,6 @@ module Users
 
     def call
       %i[
-        ensure_user_limit_not_reached!
         register_invited_user
         register_ldap_user
         ensure_registration_allowed!
@@ -65,15 +64,6 @@ module Users
       end
     end
 
-    ##
-    # Ensure the user limit is not reached
-    def ensure_user_limit_not_reached!
-      if OpenProject::Enterprise.user_limit_reached?
-        OpenProject::Enterprise.send_activation_limit_notification_about user
-        ServiceResult.new(success: false, result: user, message: I18n.t(:error_enterprise_activation_user_limit))
-      end
-    end
-
     ##
     # Try to register an invited user
     # bypassing regular restrictions
