diff --git a/app/models/queries/work_packages/columns/relation_column.rb b/app/models/queries/work_packages/columns/relation_column.rb
index 3ae476c6be..64849a5347 100644
--- a/app/models/queries/work_packages/columns/relation_column.rb
+++ b/app/models/queries/work_packages/columns/relation_column.rb
@@ -31,9 +31,4 @@
 class Queries::WorkPackages::Columns::RelationColumn < Queries::WorkPackages::Columns::WorkPackageColumn
   attr_accessor :type
 
-  def self.granted_by_enterprise_token
-    EnterpriseToken.allows_to?(:work_package_query_relation_columns)
-  end
-
-  private_class_method :granted_by_enterprise_token
 end
