diff --git a/app/models/queries/work_packages/columns/relation_of_type_column.rb b/app/models/queries/work_packages/columns/relation_of_type_column.rb
index 088b4ddbfa..cb01e36847 100644
--- a/app/models/queries/work_packages/columns/relation_of_type_column.rb
+++ b/app/models/queries/work_packages/columns/relation_of_type_column.rb
@@ -50,8 +50,6 @@ class Queries::WorkPackages::Columns::RelationOfTypeColumn < Queries::WorkPackag
   end
 
   def self.instances(_context = nil)
-    return [] unless granted_by_enterprise_token
-
     Relation::TYPES.map { |_key, type| new(type) }
   end
 end
