diff --git a/app/helpers/security_badge_helper.rb b/app/helpers/security_badge_helper.rb
index bc04f6ecdc..125eea8066 100644
--- a/app/helpers/security_badge_helper.rb
+++ b/app/helpers/security_badge_helper.rb
@@ -36,7 +36,6 @@ module SecurityBadgeHelper
       version: OpenProject::VERSION.to_semver,
       db: ActiveRecord::Base.connection.adapter_name.downcase,
       lang: User.current.try(:language),
-      ee: EnterpriseToken.current.present?,
     }.merge(args.symbolize_keys)
     uri.query = info.to_query
     uri.to_s
