diff --git a/app/models/setting.rb b/app/models/setting.rb
index 28acb81249..f886188f3b 100644
--- a/app/models/setting.rb
+++ b/app/models/setting.rb
@@ -248,11 +248,6 @@ class Setting < ApplicationRecord
 
     value = cached_or_default(name)
 
-    case name
-    when "work_package_list_default_highlighting_mode"
-      value = "none" unless EnterpriseToken.allows_to? :conditional_highlighting
-    end
-
     value
   end
 
