diff --git a/modules/ldap_groups/app/controllers/ldap_groups/synchronized_filters_controller.rb b/modules/ldap_groups/app/controllers/ldap_groups/synchronized_filters_controller.rb
index 0ff761842a..3481ef410f 100644
--- a/modules/ldap_groups/app/controllers/ldap_groups/synchronized_filters_controller.rb
+++ b/modules/ldap_groups/app/controllers/ldap_groups/synchronized_filters_controller.rb
@@ -1,7 +1,6 @@
 module LdapGroups
   class SynchronizedFiltersController < ::ApplicationController
     before_action :require_admin
-    before_action :check_ee
     before_action :find_filter, except: %i[new create]
 
     layout 'admin'
@@ -65,13 +64,6 @@ module LdapGroups
       render_404
     end
 
-    def check_ee
-      unless EnterpriseToken.allows_to?(:ldap_groups)
-        render template: 'ldap_groups/synchronized_groups/upsale'
-        return false
-      end
-    end
-
     def permitted_params
       params
         .require(:synchronized_filter)
