diff --git a/modules/ldap_groups/app/controllers/ldap_groups/synchronized_groups_controller.rb b/modules/ldap_groups/app/controllers/ldap_groups/synchronized_groups_controller.rb
index 1a13a31eb8..ffe63abc23 100644
--- a/modules/ldap_groups/app/controllers/ldap_groups/synchronized_groups_controller.rb
+++ b/modules/ldap_groups/app/controllers/ldap_groups/synchronized_groups_controller.rb
@@ -1,7 +1,6 @@
 module LdapGroups
   class SynchronizedGroupsController < ::ApplicationController
     before_action :require_admin
-    before_action :check_ee
     before_action :find_group, only: %i(show destroy_info destroy)
 
     layout 'admin'
@@ -56,13 +55,6 @@ module LdapGroups
       render_404
     end
 
-    def check_ee
-      unless EnterpriseToken.allows_to?(:ldap_groups)
-        render template: 'ldap_groups/synchronized_groups/upsale'
-        return false
-      end
-    end
-
     def permitted_params
       params
         .require(:synchronized_group)
