diff --git a/modules/ldap_groups/app/workers/ldap_groups/synchronization_job.rb b/modules/ldap_groups/app/workers/ldap_groups/synchronization_job.rb
index cda0d81a45..e5e30c497c 100644
--- a/modules/ldap_groups/app/workers/ldap_groups/synchronization_job.rb
+++ b/modules/ldap_groups/app/workers/ldap_groups/synchronization_job.rb
@@ -34,7 +34,6 @@ module LdapGroups
     self.cron_expression = '*/30 * * * *'
 
     def perform
-      return unless EnterpriseToken.allows_to?(:ldap_groups)
 
       User.system.run_given do
         synchronize!
