diff --git a/modules/two_factor_authentication/app/controllers/two_factor_authentication/two_factor_settings_controller.rb b/modules/two_factor_authentication/app/controllers/two_factor_authentication/two_factor_settings_controller.rb
index 32c7ae8246..81faeea598 100644
--- a/modules/two_factor_authentication/app/controllers/two_factor_authentication/two_factor_settings_controller.rb
+++ b/modules/two_factor_authentication/app/controllers/two_factor_authentication/two_factor_settings_controller.rb
@@ -3,7 +3,6 @@ module ::TwoFactorAuthentication
 
     before_action :require_admin
     before_action :check_enabled
-    before_action :check_ee
 
     layout 'admin'
     menu_item :two_factor_authentication
@@ -49,12 +48,6 @@ module ::TwoFactorAuthentication
       render_403 unless manager.configurable_by_ui?
     end
 
-    def check_ee
-      unless EnterpriseToken.allows_to?(:two_factor_authentication)
-        render template: 'two_factor_authentication/upsale'
-      end
-    end
-
     def manager
       ::OpenProject::TwoFactorAuthentication::TokenStrategyManager
     end
