diff --git a/app/helpers/homescreen_helper.rb b/app/helpers/homescreen_helper.rb
index 056029feac..aae4adbedd 100644
--- a/app/helpers/homescreen_helper.rb
+++ b/app/helpers/homescreen_helper.rb
@@ -37,7 +37,7 @@ module HomescreenHelper
   ##
   # Homescreen organization icon
   def organization_icon
-    op_icon('icon-context icon-enterprise')
+    op_icon('icon-context')
   end
 
   ##
@@ -55,7 +55,7 @@ module HomescreenHelper
   ##
   # Determine whether we should render the links on homescreen?
   def show_homescreen_links?
-    EnterpriseToken.show_banners? || OpenProject::Configuration.show_community_links?
+    OpenProject::Configuration.show_community_links?
   end
 
   ##
