diff --git a/app/views/ldap_auth_sources/_form.html.erb b/app/views/ldap_auth_sources/_form.html.erb
index 1ed4d10bb3..ca35adea94 100644
--- a/app/views/ldap_auth_sources/_form.html.erb
+++ b/app/views/ldap_auth_sources/_form.html.erb
@@ -137,9 +137,9 @@ See docs/COPYRIGHT.rdoc for more details.
     <%= f.check_box 'onthefly_register', label: AuthSource.human_attribute_name(:onthefly) %>
     <div class="form--field-instructions">
       <p><%= t('ldap_auth_sources.attribute_texts.onthefly_register') %></p>
-      <% if EnterpriseToken.allows_to?(:ldap_groups) %>
+
         <p><%= t('ldap_auth_sources.attribute_texts.onthefly_group_sync') %></p>
-      <% end %>
+
     </div>
   </div>
 </fieldset>
