diff --git a/modules/openid_connect/app/controllers/openid_connect/providers_controller.rb b/modules/openid_connect/app/controllers/openid_connect/providers_controller.rb
index 9fadc28776..59d43c191b 100644
--- a/modules/openid_connect/app/controllers/openid_connect/providers_controller.rb
+++ b/modules/openid_connect/app/controllers/openid_connect/providers_controller.rb
@@ -4,7 +4,6 @@ module OpenIDConnect
     menu_item :plugin_openid_connect
 
     before_action :require_admin
-    before_action :check_ee
     before_action :find_provider, only: [:edit, :update, :destroy]
 
     def index; end
@@ -54,13 +53,6 @@ module OpenIDConnect
 
     private
 
-    def check_ee
-      if EnterpriseToken.show_banners?
-        render template: '/openid_connect/providers/upsale'
-        return false
-      end
-    end
-
     def create_params
       params.require(:openid_connect_provider).permit(:name, :display_name, :identifier, :secret)
     end
