diff --git a/app/models/queries/work_packages/columns/relation_to_type_column.rb b/app/models/queries/work_packages/columns/relation_to_type_column.rb
index be0bb13582..abcd054d6f 100644
--- a/app/models/queries/work_packages/columns/relation_to_type_column.rb
+++ b/app/models/queries/work_packages/columns/relation_to_type_column.rb
@@ -46,9 +46,7 @@ class Queries::WorkPackages::Columns::RelationToTypeColumn < Queries::WorkPackag
   end
 
   def self.instances(context = nil)
-    if !granted_by_enterprise_token
-      []
-    elsif context
+    if context
       context.types
     else
       Type.all
