diff --git a/app/models/status.rb b/app/models/status.rb
index 3bf8075e84..48dfeaae70 100644
--- a/app/models/status.rb
+++ b/app/models/status.rb
@@ -88,22 +88,10 @@ class Status < ApplicationRecord
   def to_s; name end
 
   def is_readonly
-    return false unless can_readonly?
-
     super
   end
   alias :is_readonly? :is_readonly
 
-  ##
-  # Overrides cache key so that changes to EE state are reflected
-  def cache_key
-    super + '/' + can_readonly?.to_s
-  end
-
-  def can_readonly?
-    EnterpriseToken.allows_to?(:readonly_work_packages)
-  end
-
   private
 
   def check_integrity
