diff --git a/app/controllers/users_controller.rb b/app/controllers/users_controller.rb
index a79768f5b2..e7147a509b 100644
--- a/app/controllers/users_controller.rb
+++ b/app/controllers/users_controller.rb
@@ -159,13 +159,7 @@ class UsersController < ApplicationController
 
         if @user.invited?
           # setting a password for an invited user activates them implicitly
-          if OpenProject::Enterprise.user_limit_reached?
-            @user.register!
-            show_user_limit_warning!
-          else
-            @user.activate!
-          end
-
+          @user.activate!
           send_information = true
         end
 
