Index: trunk/app/controllers/issues_controller.rb
===================================================================
--- trunk/app/controllers/issues_controller.rb	(révision 1483)
+++ trunk/app/controllers/issues_controller.rb	(révision 1484)
@@ -149,7 +149,7 @@
         attach_files(@issue, params[:attachments])
         flash[:notice] = 'Demande créée avec succès'
         Mailer.deliver_issue_add(@issue) if Setting.notified_events.include?('issue_added')
-        redirect_to :controller => 'issues', :action => 'show', :id => @issue,  :project_id => @project
+        redirect_to :controller => 'issues', :action => 'show', :id => @issue
         return
       end		
     end	
